src/main/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/importering/StatistikkImportService.java

```diff
@@ -25,33 +27,92 @@ public StatistikkImportService(
     this.statistikkImportRepository = statistikkImportRepository;
   }
 
-  // TODO: DELETE ME --> bare til versifisering
```

Comment:
Gjelder ikke denne TODOen lenger?

Reply:
Jo, den gjør det, men ble borte i en refactoring. 
Men skulle vi kanskje ha noen endepunkter for å nettopp kunne verifisere?